RFC/src/RFC-0241_AtomicSwapXMR.md

```diff
@@ -0,0 +1,830 @@
+* [RFC-0202: TariScript Opcodes](RFC-0202_TariScriptOpcodes.md)
+
+
+$$
+\newcommand{\script}{\alpha} % utxo script
+\newcommand{\input}{ \theta }
+\newcommand{\cat}{\Vert}
+\newcommand{\so}{\gamma} % script offset
+\newcommand{\hash}[1]{\mathrm{H}\bigl({#1}\bigr)}
+$$
+
+
+## Description
+
+Doing atomic swaps with Monero is more complicated and requires a cryptographic dance to complete as Monero does not
+implement any form of HTLC's or the like. This means that when doing an atomic swap with Monero, most of the logic will
+have to be implemented on the Tari side. Atomic swaps between Monero and bitcoin have been implemented by the [Farcaster  project](https://github.com/farcaster-project/RFCs)
+and the [comit team](https://github.com/comit-network/xmr-btc-swap). Due to the way that TariScript works, we have a few
+advantages over the bitcoin script when it comes to [adaptor signatures](https://tlu.tarilabs.com/cryptography/introduction-to-scriptless-scripts#adaptor-signatures) as the [script key] was explicitly designed with [scriptless scripts](https://tlu.tarilabs.com/cryptography/introduction-to-scriptless-scripts) in mind.
+
+## Method
+
+The primary, happy path outline of a Tari - Monero atomic swap is described here, and more detail will follow. We assume
+that Alice wants to trade her XTR for Bob's XMR.
+
+* Negotiation - Here, both parties negotiate about the values and how the Monero and Tari UTXO's will look
+* Commitment - Here, both parties commit to their use of keys, nonces, and transactions
+* XTR payment - Here, the XTR payment is made to a special UTXO containing a script
+* XMR Payment - The Monero payment is made to a multiparty [scriptless script](https://tlu.tarilabs.com/cryptography/introduction-to-scriptless-scripts) UTXO.
+* Claim XTR - Here, the XTR is claimed, and in claiming it, the XMR private key is revealed
+* Claim XMR - Here, the XMR is claimed with the revealed key.
+
+Please take note of the notation used in [TariScript] and specifically notation used on the signatures on the [transaction inputs](RFC-0201_TariScript.md#transaction-input-changes) and on the signatures on the [transaction outputs](RFC-0201_TariScript.md#transaction-output-changes),
+other notation will be noted in the [Notation](#notation) section.
+
+## TL;DR
+
+The scheme revolves around Alice, who wants to exchange her Tari for Bob's Monero. Because they don't trust each other, 
+they have to commit some information to do the exchange. And if something goes wrong here, we want to ensure that we can 
+refund both parties either in Monero or Tari.
+
+How this works is that Alice and Bob create a shared output on both chains. The Monero output is a simple aggregate key 
+to unlock the UTXO, while multiple keys are needed to unlock the Tari UTXO. An aggregate key locks this Monero UTXO that 
+neither Alice nor Bob knows, but they both know half of the key. The current Tari block height determines the unlocking 
+key.
```

Comment:
```suggestion
key for the Tari UTXO.
```
It is ambiguous which key is being referred to in this sentence because it comes right after a sentence talking about the Monero key